eo-phi-normalizer/src/Language/EO/Phi/Rules/Yaml.hs

```diff
@@ -58,8 +60,7 @@ data Rule = Rule
 data RuleTest = RuleTest
   { name :: String
   , input :: Object
-  , output :: Object
-  , matches :: Bool
+  , output :: [Object]
```

Comment:
I understand the motivation, but do we actually have a rule where more than output is possible?
This is not even about confluence, it's just about a single rule.

Reply:
Yes, we do. Rule 1 produces two outputs.